diff --git a/hailort/cmake/external/json.cmake b/hailort/cmake/external/json.cmake
index 8c1d4b3..26211bf 100644
--- a/hailort/cmake/external/json.cmake
+++ b/hailort/cmake/external/json.cmake
@@ -4,9 +4,6 @@ include(FetchContent)

 FetchContent_Declare(
     json
-    GIT_REPOSITORY https://github.com/ArthurSonzogni/nlohmann_json_cmake_fetchcontent.git
-    GIT_TAG 391786c6c3abdd3eeb993a3154f1f2a4cfe137a0 # Version 3.9.1
-    GIT_SHALLOW TRUE
     SOURCE_DIR ${HAILO_EXTERNAL_DIR}/json-src
     SUBBUILD_DIR ${HAILO_EXTERNAL_DIR}/json-subbuild
 )
