diff --git a/hailort/cmake/external/xxhash.cmake b/hailort/cmake/external/xxhash.cmake
index 875bb19..60c1cb2 100644
--- a/hailort/cmake/external/xxhash.cmake
+++ b/hailort/cmake/external/xxhash.cmake
@@ -4,9 +4,6 @@ include(FetchContent)
 
 FetchContent_Declare(
     xxhash
-    GIT_REPOSITORY https://github.com/Cyan4973/xxHash
-    GIT_TAG bbb27a5efb85b92a0486cf361a8635715a53f6ba # Version 0.8.2
-    GIT_SHALLOW TRUE
     SOURCE_DIR ${HAILO_EXTERNAL_DIR}/xxhash-src
     SUBBUILD_DIR ${HAILO_EXTERNAL_DIR}/xxhash-subbuild
 )
